src/main/java/school/lemon/changerequest/java/introduction/hw1/Task3.java

```diff
@@ -1,7 +1,11 @@
 package school.lemon.changerequest.java.introduction.hw1;
 
 public class Task3 {
-    public static boolean isBetween(int a, int b, int c) {
-        return false;
-    }
-}
+    public static Boolean isBetween(int a, int b, int c) {
+        if (c==0){
+            return false;}
+            else {
+            return true;}
```

Comment:
For sure it's not correct solution. It do pass all tests, but what if the input will be: `2`,`1`,`0` ?
It should return `true`, but with your solution it will return `false`.